sources/presentation/Xenko.Core.Presentation/ValueConverters/ObjectToTypeName.cs

```diff
@@ -17,13 +22,87 @@ public class ObjectToTypeName : OneWayValueConverter<ObjectToTypeName>
         /// <summary>
         /// The string representation of the type of a null object
         /// </summary>
-        public const string NullObjectType = "(None)";
+        public const string NullObjectType = "(None)";        
 
         /// <inheritdoc/>
         [NotNull]
         public override object Convert(object value, Type targetType, object parameter, CultureInfo culture)
         {
-            return value?.GetType().Name ?? NullObjectType;
+            if (value == null) return NullObjectType;
+
+            return value.GetType().ToSimpleCSharpName();
+            
```

Comment:
Please remove empty line before brace

Reply:
Did you mean just delete line 34?

🤔 I could also change it to be more like the original.

```cs
return value?.GetType().ToSimpleCSharpName() ?? NullObjectType;
```

Reply:
Ha yes, even better.

Reply:
I have done this.